administrator/components/com_patchtester/config.xml

```diff
@@ -21,29 +37,55 @@
 			label="COM_PATCHTESTER_FIELD_REPO_LABEL"
 			description="COM_PATCHTESTER_FIELD_REPO_DESC"
 			default="joomla-cms"
+			id="repo"
+			showon="repository:custom"
 		/>
 
+	</fieldset>
+
+	<fieldset
+		name="component"
```

Comment:
i notice, for instance, the ftp password field doesn't auto popuplate, don't know why yet.

will check